### DISCUSSION TITLE
Headers not formatted on my machine?

### DISCUSSION THREAD

**Discussion by erdnalabs** [BryanJenksCommunity/FAQ#129] [Q&A] [answered]
Are my headers not correctly formatted?

![Screenshot 2021-12-03 at 10 47 36](https://user-images.githubusercontent.com/49495734/144590456-21fabbcf-7c2a-4701-aec5-f07b22be48a9.png)

If they are correct I would love to know how to set it up like your updated header style:

![Screenshot 2021-12-03 at 10 50 00](https://user-images.githubusercontent.com/49495734/144590500-1eafdd7f-6f68-4985-99c1-83ddf5295868.png)

Best regards

**Comment by tallguyjenks** [comment, marked as answer]
your image reflects what i currently have, the older style is different CSS that is commented out in the file you'd have to comment out the part with the current headings CSS and uncomment the old version
